src/assertions/elValueContains.js

```diff
@@ -13,7 +13,7 @@ util.inherits(ElValueContains, BaseAssertion);
 
 ElValueContains.prototype.assert = function (actual, expected) {
   if (expected === undefined
-    || actual.indexOf(expected) < 0
+    || !actual || actual.indexOf(expected) < 0
```

Comment:
what is the use case to this logic here, `elValueContains()` without any param is expected to fail?